.github/workflows/all_builds.yml

```diff
@@ -1,7 +1,7 @@
 name: 🌈 All Builds
 on:
   push:
-    branches: [ master ]
+    branches: [ master, godot-4.0-adaption ]
```

Comment:
could we also add godot-4.0 here, too?

Reply:
Sure, I'll add it.
When godot-4.0-adaption gets merged, this will be cleaned up most likely.

Reply:
Correct!